pkg/config/stepmeta_test.go

```diff
@@ -262,7 +262,7 @@ func TestGetContextParameterFilters(t *testing.T) {
 		assert.Equal(t, []string{"testSecret1", "testSecret2"}, filters.Steps, "incorrect filter Steps")
 		assert.Equal(t, []string{"testSecret1", "testSecret2"}, filters.Stages, "incorrect filter Stages")
 		assert.Equal(t, []string{"testSecret1", "testSecret2"}, filters.Parameters, "incorrect filter Parameters")
-		assert.Equal(t, []string{"testSecret1", "testSecret2"}, filters.Env, "incorrect filter Env")
```

Comment:
Not sure of that is related to env in the sense of os env. I guess here it is about the metadata.